components/clm/bld/namelist_files/use_cases/2015-2100_SSP585_transient.xml

```diff
@@ -15,33 +15,20 @@
 
 <clm_demand >flanduse_timeseries</clm_demand>
 
-<stream_year_first_ndep phys="clm4_0" bgc="cn"   >1850</stream_year_first_ndep>
-<stream_year_last_ndep  phys="clm4_0" bgc="cn"   >2005</stream_year_last_ndep>
-<model_year_align_ndep  phys="clm4_0" bgc="cn"   >1850</model_year_align_ndep>
-
-<stream_year_first_ndep phys="clm4_0" bgc="cndv" >1850</stream_year_first_ndep>
-<stream_year_last_ndep  phys="clm4_0" bgc="cndv" >2005</stream_year_last_ndep>
-<model_year_align_ndep  phys="clm4_0" bgc="cndv" >1850</model_year_align_ndep>
-
+<!-- *** NOT USED FOR SATELLITE PHENOLOGY ***
```

Comment:
Is there a reason for keeping this entries as commented out instead of deleting them?

Reply:
@cameronsmith1, Any comment on my question above?

Reply:
The diffs by Github are confusing for this file.  The clm4.0 and 5.0 lines are deleted.  The clm4.5 lines are commented out -- because they will probably be needed by the BGC version of this compset.